pset-functions.js

```diff
@@ -208,6 +332,28 @@ console.log('Problem 5 - bonus:')
 
 // Add your code below this line
 
+function wordCount(phrase) {
+  let words = phrase.replace(',', '').split(' ')
+  console.log(words)
+  const count = {}
+  words.forEach((word) => {
```

Comment:
Really awesome use of the forEach method; and great use of the if/else statement to add key/value pairs to the empty object!

Natalie, you did a really great job here! Keep up the good work :)